src/In.ProjectEKA.HipService/Link/Database/Migrations/20200707040844_UnlinkedAccounts.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System;
+using Microsoft.EntityFrameworkCore.Migrations;
+using Npgsql.EntityFrameworkCore.PostgreSQL.Metadata;
+
+namespace In.ProjectEKA.HipService.Link.Database.Migrations
+{
+    public partial class UnlinkedAccounts : Migration
```

Comment:
Codacy found an issue: ['partial' is gratuitous in this context.](https://app.codacy.com/gh/ProjectEKA/hip-service/pullRequest?prid=5811615)